index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=collectorsapp', 'root', '');
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$sql = 'SELECT * FROM `grocery_item`';
+$sql = $db->prepare('SELECT * FROM `grocery_item`;');
```

Comment:
I would suggest putting your query -> execute -> output process into a function, rather than inline.

Reply:
ok did it thanks